### ISSUE TITLE
v1.7.8 on MacOS: Unable to start application, hangs on verification from system

### ISSUE THREAD

**Issue by alexander-daniel** [plotly/falcon-sql-client#247]
Looks awesome, can't wait to try it!

I dragged the application to my `Applications` folder after unzipping. When I tried to open Falcon, it just hangs at the following:
![screen shot 2017-11-14 at 10 56 54 am](https://user-images.githubusercontent.com/5408720/32789789-b1cc5eaa-c92a-11e7-9a28-61dd5f9847cb.png)

Thanks!

**Comment by n-riesco** [comment]
v1.7.8 isn't the latest version.
Have you tried the latest release, [v2.3.2](https://github.com/plotly/falcon-sql-client/releases/download/v2.3.2/mac-falcon-sql-client.zip)?

**Comment by alexander-daniel** [comment]
Thanks for the quick reply @n-riesco 

So I clicked the "Download for Mac" button on the falcon homepage
![screen shot 2017-11-14 at 2 13 45 pm](https://user-images.githubusercontent.com/5408720/32799537-106d5796-c946-11e7-9be9-b40ca0d8d4f7.png)

That links to what appears to be the proper release link on github for `2.3.2`
`https://github.com/plotly/falcon-sql-client/releases/download/v2.3.2/mac-falcon-sql-client.zip`

But when I unzip the archive and open up the `version` below
![screen shot 2017-11-14 at 2 13 16 pm](https://user-images.githubusercontent.com/5408720/32799582-2d17eb7c-c946-11e7-8e63-779dfbcf2bd4.png)

it reads: `1.7.8`

**Comment by michaelschoellhammer** [comment]
i have the same issue. how can i get the latest release?

**Comment by n-riesco** [comment]
PR #277 is addressing this issue. We can update the installers once it gets merged.

If you are familiar with Node.js development, these are the instructions to build the app:
```
git clone https://github.com/plotly/falcon-sql-client.git
cd falcon-sql-client
yarn install
yarn build
yarn run rebuild:modules:electron
```
and to launch the app:
```
yarn start
```

**Comment by n-riesco** [comment]
Closed via #277
